### ISSUE TITLE
Is it still maintained?

### ISSUE THREAD

**Issue by anasmano** [rekab-app/background_locator#370]
I'm hesitant to use this library in production, because it hasn't been updated for a long time, considering that flutter and dart versions are developing very quickly.

**Comment by OlivierBournival** [comment]
no,  it does not appear to be, go to background_locator_2 its more recent. Or use https://pub.dev/packages/location , 
 instead
